src/android/io/cozy/plugins/listlibraryitems/ListLibraryItems.java

```diff
@@ -217,8 +234,16 @@ private boolean listItems(CallbackContext callbackContext, boolean includePictur
         }
 
         // https://stackoverflow.com/a/20429397
-        final String selection = MediaStore.Images.Media.BUCKET_DISPLAY_NAME + " = ?";
-        final String[] selectionArgs = new String[] { "Camera" };
+        String selection = MediaStore.Images.Media.BUCKET_DISPLAY_NAME + " IN ( ";
+        final String[] selectionArgs = new String[mediaBuckets.length()];
+        for (int i = 0; i < mediaBuckets.length(); i++) {
+            selection += "?";
+            if(i != mediaBuckets.length() - 1) {
+                selection += ", ";
+            }
+            selectionArgs[i] = mediaBuckets.optString(i);
+        }
+        selection += " ) ";
```

Comment:
Awesome! I were a bit worried about having to make a request for every Buckets! 👍